ext/node/polyfills/fs.ts

```diff
@@ -165,6 +169,7 @@ const promises = {
   opendir: opendirPromise,
   rename: renamePromise,
   truncate: truncatePromise,
+  ftruncate: ftruncatePromise,
```

Comment:
Node doesn't seem having `fs.promises.ftruncate` (No doc entry for it, and also not available in runtime) https://nodejs.org/api/fs.html

Let's not expose it

Reply:
You make a great point! I've noticed that `fstat` is also exposed but can't seem to find it in the doc, and `fs.promises.fstat` also doesn't seem to be available in the runtime. Is that an issue, and if so, is it something we would like to handle within this PR?

Reply:
> fstat is also exposed but can't seem to find it in the doc, and fs.promises.fstat also doesn't seem to be available in the runtime. Is that an issue

Good point. Yes, that seems an issue to me.

>, and if so, is it something we would like to handle within this PR?

Fixing it in this PR is welcome, but that's not absolutely necessary in my opinon. I created an issue for `fs.promises.fstat` for now https://github.com/denoland/deno/issues/27423

Reply:
Awesome, I might take a deeper look into that one as well :)

Also, the implementation for `promises.ftruncate` has been cleaned up 🧹